extensions/docuSign/actions/createEmbeddedSignatureRequestWithTemplate/createEmbeddedSignatureRequestWithTemplate.ts

```diff
@@ -80,7 +80,7 @@ export const createEmbeddedSignatureRequestWithTemplate: Action<
         userName: signerName,
         clientUserId: patientId,
         returnUrl: replaceStringVariables(returnUrlTemplate, {
-          sessionId,
```

Comment:
is this change relevant in his MR? this might be one that Radoslaw made, but if we can move this to a separate pull request, that would be good

Reply:
@michal-grzelak can this go in another PR?

Reply:
What happens in here I believe is that after updating `@awell-health/extensions-core` to new version the type of `sessionId` that is received in action payload changed from `string` to `string | undefined`, whereas this function expects `string`. This means that without this cast, Typescript throws error and tests will fail, so unfortunately from all external changes, this must be included.